### ISSUE TITLE
Create an accessibility guild email address that's publicly accessible

### ISSUE THREAD

**Issue by maya** [18F/accessibility#239]
We currently list the email `g-accessibility@gsa.gov` as a contact on the Accessibility for Teams site, but members of the public are unable to contact us through it because it's only accessible to guild members. 

We should have an email that members of the public can reach us at (guild leads and a11y project members).

/cc @davidstenger -- couldn't assign you, are you a member of this repo?

**Comment by nkkl** [comment]
Currently discussing with Andre.

**Comment by nkkl** [comment]
Go ahead and use `tts-accessibility@gsa.gov`!
